docs/specs/sections/protocol.tex

```diff
@@ -0,0 +1,105 @@
+
+In Citadel, each party involved in the protocol keeps static keys, as we detail now. Let $G, G'\gets\mathbb{J}$ be two generators for the subgroup $\mathbb{J}$ of order $t$ of the Jubjub. The keys of each party as the following.
+
+\begin{itemize}%[label=$\bullet$]
+	\item \textit{Secret key:} $\sk = (a,b)$, where $a,b\gets \F_t$.
+	\item \textit{Public key:} $\pk = (A,B)$, where $A = a G$ and $B = b G$.
+\end{itemize}
+
+The workflow of the Citadel protocol is depicted in Figure \ref{fig:protocol}, and described with full details as follows.
+
+\begin{figure}[h]
+	\centering
+		\includegraphics[width=390pt,draft=false]{images/protocol.eps}
```

Comment:
I think that the figure should reflect that LP and SP can be different entities, and point the arrows to the right entity.

Reply:
As we discussed, in a future version of the specs we will include how the user interacts with the contract and will make this distinction.